assessment__new-hire--app-dev/src/store.js

```diff
@@ -2,72 +2,105 @@ import Vue from 'vue'
 import Vuex from 'vuex'
 import axios from 'axios'
 import uuidv1 from 'uuid/v1';
+const ans = require('../../Backend/db/api_call');
+// const r = require('../../Backend/app');
+
 
 Vue.use(Vuex)
 
 import { API_ENDPOINT } from './config';
-import testData from '@/data/test-data';
+let temp = (`${API_ENDPOINT}/peek`);
+
+
 
 export default new Vuex.Store({
+    
     state: {
         // TODO don't use testData
-        events: testData,//{},
+
+        events: [],
         basicToken: false
     },
     mutations: {
         submitBasicToken: function( state, token ) {
             state.basicToken = `basic ${token}`;
         },
         updateList: function( state, list ) {
-            state.events = list;
+            let tempArray = [];
+            let finalJson = [];
```

Comment:
lets talk through the logic of this, if this is for ui data  formatting is there a better way to do it?